index.html

```diff
@@ -91,52 +168,62 @@ <h2 class="myh24">About me</h2>
-            <li>CapyBara</li>
-            <li>Selenium</li>
-          </ul>
-          <ul class="ul3">
-            <li>&nbsp;Skills</li>
-          </ul>
-          <ul class="childitems">
-            <li>Codekit</li>
-            <li>Github</li>
-            <li>Codepen</li>
-            <li>Gitlab</li>
-            <li>Terminal</li>
-          </ul>
+            <ul class="childitems">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>Css</li>
+            </ul>
+          </div>
+          <div class="ul2wrapper">
+            <ul class="ul2">
+              <li>&nbsp;Frameworks</li>
+            </ul>
+            <ul class="childitems">
+              <li>Bootstrap</li>
+              <li>Ruby on Rails</li>
+              <li>RSPec</li>
+              <li>CapyBara</li>
+              <li>Selenium</li>
+            </ul>
+          </div>
+          <div class="ul3wrapper">
+            <ul class="ul3">
+              <li>&nbsp;Skills</li>
+            </ul>
+            <ul class="childitems">
+              <li>Codekit</li>
+              <li>Github</li>
+              <li>Codepen</li>
+              <li>Gitlab</li>
+              <li>Terminal</li>
+            </ul>
+          </div>
         </div>
       </section>
     </div>
     <div id="form-section">
-      <p id="form-title">I'm always interested in hearing about new projects,
-      so if you'd like to chat please get in touch.
-      </p>
-      <form id="myform" action="https://formspree.io/f/mknyzadl" method="POST">
-        <input class="formitem" type="text" id="fullname" name="fullname" placeholder="Full Name" required maxlength=30>
-        <hr class="hr3">
-        <input class="formitem" type="email" id="email" name="email" placeholder="Email Address" required>
-        <hr class="hr3">
-        <textarea class="formitem" id="message" name="message" required placeholder="Enter text here" maxlength=500></textarea>
-        <div id="buttonwrapper">
-          &nbsp;<button class="button" type="submit">Get in Touch</button>&nbsp;
+      <div id="horizvert">
+        <p id="form-title">I'm always interested in hearing about new projects,
+        so if you'd like to chat please get in touch.
+        </p>
+        <div id="formproper">
+          <form id="myform" action="https://formspree.io/f/mknyzadl" method="POST">
+            <input class="formitem" type="text" id="fullname" name="fullname" placeholder="Full Name" required maxlength=30>
+            <hr class="hr3">
+            <input class="formitem" type="email" id="email" name="email" placeholder="Email Address" required>
+            <hr class="hr3">
+            <textarea class="formitem" id="message" name="message" required placeholder="Enter text here" maxlength=500></textarea>
+            <div id="buttonwrapper">
+              &nbsp;<button class="button" type="submit">Get in Touch</button>&nbsp;
+            </div>
+          </form>
```

Comment:
- [ ] Please add the correct width value to the form elements in the desktop version and try to stick as much as possible to the given Figma design.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/160032335-59c25ef9-4adb-4096-abc9-b55309c11630.png)  | ![image](https://user-images.githubusercontent.com/68796232/160032432-3aa59bc6-e5ac-4474-a848-87bbc2b907c7.png)  |

Reply:
I can't understand why the images you provide are different widths. I look in my browser and figma design and I get this:
![2022-03-24](https://user-images.githubusercontent.com/44879977/160042785-bf317f90-3426-4e10-8d5f-56925573b92e.jpg)
![2022-03-24 figma](https://user-images.githubusercontent.com/44879977/160042830-516f8078-1ab7-439b-a698-d62fd2041fc3.jpg)

Reply:
Here I can't find anything to correct :pensive:

Reply:
Finally, I changed the width of the form from 100% to 50%, it can make a difference in other browsers than mine (Chrome), and it looks better.